report/src/main/webapp/app/components/keyboardShortcuts.directive.js

```diff
@@ -208,13 +216,64 @@ define(['angularAMD', 'userSettingsService'], function (angularAMD) {
         }
       }
 
-      function toggleNextTest(currentTest) {
-        $(currentTest).nextAll('.aside-report:not(.is-hidden)')
-        .first()
-        .find('.test-name')
-        .click();
+      function handleStyling(testContainer, currentItem, $element) {
+        testContainer.removeClass('is-active');
+        currentItem.removeClass('is-active');
+        $element.addClass('is-expanded');
+        $element.children().first().addClass('is-active');
+        scrollTo($element.find('.is-active'));
+      }
+
+      function toggleNextTestItem(currentTest) {
+        $(currentTest).nextAll(
+          '.aside-report:not(.is-hidden)')
+          .first()
+          .find('.test-url')
+          .first()
+          .click();
       }
 
+      function togglePrevTestItem(currentTest) {
+        $(currentTest).prevAll(
+          '.aside-report:not(.is-hidden)')
+          .first()
+          .find('.test-url')
+          .last()
+          .click();
+      }
+
+      function clickOnTab() {
+        var currentTabLabel = userSettingsService.getLastTab();
+        var $nextTestTabs = $('.nav-tabs').children();
```

Comment:
In some places you use multiple `var` notation (here) and in others one `var` and variable declaration separated out by a comma. I'd be good to stick with just one of them.

Reply:
Changed, but I still got some stuff to do (with MutationObserver) so I will commit tomorrow.